tests/dragonfly/utility.py

```diff
@@ -364,7 +364,7 @@ def __init__(
         if cluster_mode:
             max_multikey = 1
             multi_transaction_probability = 0
-            unsupported_types = [ValueType.JSON]  # Cluster aio client doesn't support JSON
+            unsupported_types.append(ValueType.JSON)  # Cluster aio client doesn't support JSON
```

Comment:
Actually, why do we disable it for all other tests only because of cluster?

Reply:
only for cluster - if cluster_mode:

Reply:
Oh sorry, I'm blind 🙈